releases/release-8.4.0.md

```diff
@@ -0,0 +1,546 @@
+
+    Traffic replay is useful in the following scenarios:
+
+    - Validate TiDB version upgrades
+    - Assess change impact
+    - Validate performance before scaling TiDB
+    - Test performance limits
+
+    For more information, see [documentation](/tiproxy/tiproxy-traffic-replay.md).
+
+### SQL
+
+* Support vector search (experimental) [#54245](https://github.com/pingcap/tidb/issues/54245) [#9032](https://github.com/pingcap/tiflash/issues/9032) @[breezewish](https://github.com/breezewish) @[Lloyd-Pottiger](https://github.com/Lloyd-Pottiger) @[EricZequan](https://github.com/EricZequan) @[zimulala](https://github.com/zimulala) @[JaySon-Huang](https://github.com/JaySon-Huang) @[winoros](https://github.com/winoros) @[wk989898](https://github.com/wk989898) **tw@qiancai** <!--1898-->
+
+    Vector search is a search method based on data semantics, which provides more relevant search results. As one of the core functions of AI and large language models (LLMs), vector search can be used in various scenarios such as Retrieval-Augmented Generation (RAG), semantic search, and recommendation systems.
+
+    Starting from v8.4.0, TiDB supports [vector data types](/vector-search-data-types.md) and [vector search indexes](/vector-search-index.md), offering powerful vector search capabilities. TiDB vector data types support up to 16,383 dimensions and support various [distance functions](/vector-search-functions-and-operators.md#vector-functions), including L2 distance (Euclidean distance), cosine distance, negative inner product, and L1 distance (Manhattan distance).
+
+    To start vector search, you only need to create a table with vector data types, insert vector data, and then perform a query of vector data. You can also perform mixed queries of vector data and traditional relational data.
+
+    To enhance the performance of vector search, you can create and use [vector search indexes](vector-search-index.md). Note that TiDB vector search indexes rely on TiFlash. Before using vector search indexes, make sure that TiFlash nodes are deployed in your TiDB cluster.
+
+    For more information, see [documentation](/vector-search-overview.md).
+
+### DB operations
+
+* BR supports client-side encryption of log backup data (experimental) [#55834](https://github.com/pingcap/tidb/issues/55834) @[Tristan1900](https://github.com/Tristan1900) **tw@qiancai** <!--1920-->
+
+     In earlier TiDB versions, only snapshot backup data can be encrypted on the client side. Starting from v8.4.0, log backup data can also be encrypted on the client side. Before uploading log backup data to your backup storage, you can encrypt the backup data to ensure its security via one of the following methods:
+
+    - Encrypt using a custom fixed key
+    - Encrypt using a master key stored on a local disk
+    - Encrypt using a master key managed by a Key Management Service (KMS)
```

Comment:
There are many things like KMS, RAG, IMDSv2, EC2, OOM, RU, TSO, PD, RPC that might benefit from a link to glossary.md

Reply:
@qiancai PTAL

Reply:
Yes, that's a good idea. Currently, the [Glossary](https://docs.pingcap.com/tidb/dev/glossary) doc lacks explanations for many abbreviations. To improve clarity and accessibility for users, we need to consider expanding the existing glossary to include these abbreviations alongside the product-specific terms.

Reply:
@qiancai Maybe you can create an issue for this with a list of what we know is missing? Then I can help to add explanations. I think this can be done after the v8.4 release.

Reply:
@dveeden Thanks for your kind help. Here is the issue https://github.com/pingcap/docs/issues/19212. In this issue, I've picked some frequently used abbreviations as candidates.